python/demo/demo_tnt-elements.py

```diff
@@ -68,8 +69,8 @@
 # +
 geometry = basix.geometry(basix.CellType.quadrilateral)
 topology = basix.topology(basix.CellType.quadrilateral)
-x = [[], [], [], []]
-M = [[], [], [], []]
+x: typing.List[typing.List[np.ndarray]] = [[], [], [], []]
```

Comment:
This is rather cryptic for a user (not that the previous code was pretty).

Reply:
Decided to ignore the specific typing error.